<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QDialog" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#Setting { background-color: rgb(247,250,253); }
QListWidget#listWidget { background-color: rgb(247,250,253); }
QStackedWidget#stackedWidget { background-color: rgb(255,255,255); }
QWidget#pageSetThreadNum { background-color: rgb(255,255,255); }
QWidget#pageSetFileFilters { background-color: rgb(255,255,255); }
QFrame#frame_pBug { background-color: rgb(255,255,255); }
Line { background-color: rgb(235,235,236); }</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,0,8">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="1,9">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="labelTitle">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../res/resource.qrc">:/img/image/Setting_new.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>SetThreadNum</string>
            </property>
            <property name="icon">
             <iconset resource="../res/resource.qrc">
              <normaloff>:/img/image/settinghold.png</normaloff>:/img/image/settinghold.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>SetFileFilters</string>
            </property>
            <property name="icon">
             <iconset resource="../res/resource.qrc">
              <normaloff>:/img/image/FileFilters.png</normaloff>:/img/image/FileFilters.png</iconset>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <widget class="QWidget" name="pageSetThreadNum">
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>40</y>
             <width>131</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Set Check Thread Num</string>
           </property>
          </widget>
          <widget class="QSlider" name="horSlidThreadNum">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>80</y>
             <width>391</width>
             <height>21</height>
            </rect>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>4</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>3</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
          <widget class="QLabel" name="label_ThreadNum">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>120</y>
             <width>101</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Thunder Num: %1</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="pageSetFileFilters">
          <widget class="QLineEdit" name="lineEdit">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>30</y>
             <width>291</width>
             <height>20</height>
            </rect>
           </property>
           <property name="placeholderText">
            <string>Join a custom file format</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton">
           <property name="geometry">
            <rect>
             <x>320</x>
             <y>30</y>
             <width>80</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
          <widget class="QScrollArea" name="scrollArea">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>70</y>
             <width>441</width>
             <height>231</height>
            </rect>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>427</width>
              <height>500</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>500</height>
             </size>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="Line" name="line_2">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_pBut">
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="9,1">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="spacing">
               <number>6</number>
              </property>
              <item>
               <widget class="QPushButton" name="pButOK">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>OK</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pButCancel">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>Cancel</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/resource.qrc"/>
 </resources>
 <connections/>
</ui>
